include/libtorrent/entry.hpp

```diff
@@ -95,8 +95,30 @@ namespace libtorrent
 			bool operator()(T1 const& rhs, T2 const& lhs) const
 			{ return rhs < lhs; }
 		};
-	};
+
+		template<class T>
+		class map_string : public std::map<std::string, T, aux::strview_less>
+		{
+		};
+#else
+		template<class T>
```

Comment:
since this is master, C++11 can be assumed, and this else-block should probably just be removed. In fact, just removing the conditionals from the previous version is probably even simpler